packages/server/README.md

```diff
@@ -44,6 +56,15 @@ yarn dev
 
 Runs the app in the development mode. Open [http:/localhost:3000](http:/localhost:3000) to view it in the browser.
 
+If you want to work on the [provider portal](http://localhost:3000/provider), you can log in with this google user:
+
+```
+username: dev.upswyng@gmail.com
```

Comment:
We should make a follow-up to this to make it a @codeforboulder.org email for easier management. Not needed for this PR though!

Reply:
👍